sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala

```diff
@@ -182,7 +182,11 @@ class HiveScriptTransformationSuite extends BaseScriptTransformationSuite with T
         identity,
         df.select(
           'a.cast("string").as("key"),
-          'b.cast("string").as("value")).collect())
+          concat_ws("\t",
+            'b.cast("string"),
+            'c.cast("string"),
+            'd.cast("string"),
+            'e.cast("string")).as("value")).collect())
```

Comment:
What happens if a single column given with hive serde in this case? https://github.com/apache/spark/pull/29414/files#diff-01228f8ade90c259db2dfdf31ea8a5d1R61-R63

Reply:
> What happens if a single column given with hive serde in this case? https://github.com/apache/spark/pull/29414/files#diff-01228f8ade90c259db2dfdf31ea8a5d1R61-R63

See https://github.com/apache/spark/pull/29421#issuecomment-677495619

Reply:
Oh, I see. In the case, we should return NULL.